sdk/python/generative-ai/rag/code_first/README.md

```diff
@@ -6,7 +6,7 @@ Read more about their structure [here](./docs/mlindex.md).
 ## Pre-requisites
 
 0. Install `azure-ai-ml` and `azureml-rag`:
-    - `pip install 'azure-ai-ml==1.10.0a20230825006' --extra-index-url https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/`
+    - `pip install 'azure-ai-ml==1.10'`
     - `pip install -U 'azureml-rag[document_parsing,faiss,cognitive_search]>=0.2.0'`
```

Comment:
Some notebook have pip installs with upgrade options, but some don't. Do we need to standardize it? 

Also do we need pip installs in each notebook? Could we just add it to the README.md and point every notebook to the same place?

Reply:
Having the install in the Notebooks can help as some user cases need a different set of install extras (like using faiss vs cognitive_search).

Reply:
This install does seem to cover both the faiss and cognitive_search case though so should be able to remove the install from the files.